### DISCUSSION TITLE
Declare the windows.protocol usage of the protocol/scheme `https` in your AppxManifest.xml file"

### DISCUSSION THREAD

**Discussion by dflyers** [dotnet/maui#16192] [Q&A]
Hi, 
Anyone have an example of how to set this up ? 
I have it as , 

<Extensions> <uap:Extension Category="windows.protocol"> <uap:Protocol Name="https" /> </uap:Extension> </Extensions> 

in the package.appxmanifest file. But still get the error.

thanks!

**Comment by ITaluone** [comment]
+1

How can I register my app to windows, that my app is able to handle e.g. `tel://` / `callto://` links

**Comment by damusthe** [comment]
I've the same concern with the following error
_System.InvalidOperationException : 'You need to declare the windows.protocol usage of the protocol/scheme `com.maui.login` in your AppxManifest.xml file'_

using the method
`await WebAuthenticator.AuthenticateAsync(new WebAuthenticatorOptions()`
